src/lib/containers/GenericCard.tsx

```diff
@@ -161,10 +161,10 @@ export const renderLabel = (args: {
     if (strList) {
       return strList.map((el, index) => {
         return (
-          <>
+          <span>
```

Comment:
Snuck this in here.  It's a fix for https://sagebionetworks.jira.com/browse/PORTALS-1718 (verified that not all items are underlined anymore with this change)